File: src/app/asset/asset.ts
	
export class Asset {
 
  id: number;
  name: string;
  description: string;
  status: string;
  
 
  constructor(id: number, name: string, description: string, status: string){
    this.id = id;
    this.name = name;
    this.description = description;
    this.status = status;
  }
 
}

File: sprint-boot-example/src/main/java/exampleTest/Asset.java
package exampleTest;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;
import javax.validation.constraints.NotNull;  

@Entity
@Table(name="ASSET")
public class Asset {
	@Id
	@GeneratedValue(strategy = GenerationType.AUTO)
	private int Id;
	@NotNull
	private String Name;
	@NotNull
	private String Description;
	@NotNull
	private String Status;
	
	public int getId() {
		return Id;
	}
	public void setId(int id) {
		Id = id;
	}
	
	public String getName() {
		return Name;
	}
	public void setName(String name) {
		Name = name;
	}
	public String getDescription() {
		return Description;
	}
	public void setDescription(String description) {
		Description = description;
	}
	public String getStatus() {
		return Status;
	}
	public void setStatus(String status) {
		Status = status;
	}
}
